input/fsh/SD_Genomics.fsh

```diff
@@ -110,9 +110,8 @@ Observation resources associated with an mCODE patient with an Observation.code
 
 * status and code and subject and effective[x] and value[x] and focus MS
 * subject 1..1
-* code from TumorMarkerTestVS (extensible)
+* code from TumorMarkerTestVS (required)
 * subject only Reference(CancerPatient)
-* focus only Reference(CancerConditionParent)
```

Comment:
Removing this makes it less clear how we expect `focus` to be used, which I think should be defined if `focus` is still going to be Must Support.

I believe the current description for `focus` is generic:

> What the observation is about, when it is not about the subject of record

Do you want to add in a custom description?

Reply:
I'm not sure this makes sense. If this is a blood or urine test, the test is on the patient, not on the disease. The disease is more like the reason for the test. So I actually don't think the focus should be used in this case.

Reply:
I agree, and I think we should provide some specific guidance in a description for `focus`. I'll add an attempt below.

Reply:
Max & I concluded the best course of action was to remove MS from focus.